src/mlpack/core/data/load_impl.hpp

```diff
@@ -410,6 +410,110 @@ bool LoadCategorical(const std::string& filename,
   return true;
 }
 
+template<typename MatType>
+bool Load(const std::vector<std::string>& filenames,
+          MatType& matrix,
+          const TextOptions& opts)
+{
+  TextOptions copyOpts(opts);
+  return Load(filenames, matrix, copyOpts);
+}
+
+template<typename MatType>
+bool Load(const std::vector<std::string>& filenames,
+          MatType& matrix,
+          TextOptions& opts)
+{
+  bool success = false;
+  MatType tmp;
+  arma::field<std::string> firstHeaders;
+  if (filenames.empty())
+  {
+    if (opts.Fatal())
+    {
+      Log::Fatal << "Load(): Please specify the filenames to be loaded!"
+          << std::endl;
```

Comment:
```suggestion
      Log::Fatal << "Load(): given set of filenames is empty; loading failed."
          << std::endl;
```
Up to you if you want to take the suggestion, but I think it's important to be clear about exactly what is wrong.  With the current error message, I could believe that a user might not be able to quickly figure out where they screwed up.  I'll leave another suggestion below for the `Warn` case.